reviews/review-01+2020-11-02.md

```diff
@@ -0,0 +1,305 @@
+
+-   Material that includes the human side of project development
+
+## What reservations do you have about this manuscript?
+
+I have no big-picture reservations about this manuscript. Some chapters
+would benefit from additional review or editing to further clarify their
+content; many of the exercises still need solutions, but I see these
+more as editorial concerns (addressed in the following questions) rather
+than significant structure or content reservations.
+
+## Content and structure
+
+In terms of content and structure, do you think the chapters need any
+revisions, such as additions or omissions? Please be as specific as
+possible.
+
+The opening chapter does an excellent job of setting up the audience and
+"syllabus" of the book. Its aim would be strengthened by adding a
+section on "How to Use this Book" or expanding the Syllabus (1.3) and/or
+Audience (1.2) sections with:
+
+-   Suggestions on how the book could be used. This could be as simple
+    as -- "Work through each chapter, following along with the commands
+    and changes described in the text. Try at least half of the
+    exercises for each chapter" -- to something as complex as an
+    appendix with suggestions for a suggested course plan, adapting
+    these materials to a lecture, finding a mentor or someone to help
+    you (if doing self-study), etc.
+
+-   Emphasizing the presence of exercises and solutions, as well as
+    recommendations for how to use them. I agree with the decision to
+    include the exercises at the end of each chapter, but I think a note
+    recommending that readers not wait until the end to check out some
+    of the available exercises would be a good idea.
+
+-   Outlining the standard chapter format: introduction, content,
+    exercises; identifying the callout box format as a mechanism to
+    supplement the main flow of text.
+
+My only additional comments are about rearranging/splitting specific
+material or chapters. None of these are strictly necessary -- I leave it
+up to the discretion of the authors as to whether they would improve the
+overall manuscript or not.
+
+-   The project organization description (1.4) seemed a bit out of place
+    in chapter 1 -- most of that chapter is about course **details**
+    (audience, installation, syllabus), but the Project Structure
+    section reads like course **content**. If the above suggestion is
+    added, this will even more be the case. I see a compelling argument
+    for making the current chapter 1 into a numberless prologue or
+    introduction and having chapter 1 be a very short chapter with the
+    Project Structure content.
+
+-   Chapter 2 (The Basics of the Unix Shell) is very long. Content-wise,
+    there isn't a strong reason to split (although the separation
+    between 2.8 and 2.9 is one possible place to do so); I would mainly
+    recommend splitting this chapter so that there isn't so much content
+    before getting to exercises.
+
```

Comment:
I tend to agree that The Basics of the Unix Shell is too long. I also agree that separating between 2.8 and 2.9 would be the place to do it. The three shell chapters (I'm a little scared of there being so many) would therefore cover:
- Single bash commands
- Combining multiple bash commands
- Shell scripts